### DISCUSSION TITLE
The IP Pool Option Does Not Appear.

### DISCUSSION THREAD

**Discussion by TR-SametOzen** [postalserver/postal#3141] [Development discussions]
Hello,

although I have enabled the IP Pool in my config file, I can't see it in the settings section. What should I do extra? I restarted the server.

![image](https://github.com/user-attachments/assets/aceaa3e5-f897-40fe-8a86-2eb9b285d44f)

![image](https://github.com/user-attachments/assets/31f3d682-76aa-4ca5-b1d4-f5e950fb63cb)

**Comment by willpower232** [comment]
According to the docs, the link is in the top right of the screen https://docs.postalserver.io/features/ip-pools#configuring-ip-pools

**Reply by TR-SametOzen** [reply]
Hello,

Yes, I could see it after these settings. But now I have a problem: e-mails are sent from the first IP address of the server. However, I increased the priority of the 2nd IP address.

![image](https://github.com/user-attachments/assets/4e3cfe85-064a-44d0-ad6b-521d475dd664)
